stake-pool/js/src/instructions.ts

```diff
@@ -96,6 +100,73 @@ export const STAKE_POOL_INSTRUCTION_LAYOUTS: {
       BufferLayout.ns64('poolTokens'),
     ]),
   },
+  /// (Staker only) Increase stake on a validator again in an epoch.
+  ///
+  /// Works regardless if the transient stake account exists.
+  ///
+  /// Internally, this instruction splits reserve stake into an ephemeral stake
+  /// account, activates it, then merges or splits it into the transient stake
+  /// account delegated to the appropriate validator. `UpdateValidatorListBalance`
+  /// will do the work of merging once it's ready.
```

Comment:
nit: Out of consistency, could you remove all of the comments in these instruction layouts?